microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -80,8 +80,18 @@ For more information, see xref:../microshift_support/microshift-etcd.html#micros
 //
 // This content will be added post-GA, as it is asynchronous content.
 
-//[id="microshift-4-13-networking"]
-//=== Networking
+[id="microshift-4-13-networking"]
+=== Networking
+
+[id="microshift-4-13-load-balancer"]
+==== Deploying network load balancers on {product-title}
+
+{product-title} now offers a built-in implementation of network load balancers. For more details, see xref:../microshift_networking/microshift-networking.adoc#microshift-deploying-a-load-balancer_microshift-networking[Deploying a TCP load balancer on a workload].
+
+[id="microshift-4-13-blocking-nodeport"]
+==== Blocking a NodePort service on a host interface 
+
+You can now block a NodePort service on a specific host interface and restrict access from outside a {product-title} node. For more information, see  xref:../microshift_networking/microshift-networking.html#microshift-blocking-nodeport-access_microshift-networking[Blocking external access to the NodePort service on a specific host interface].
```

Comment:
Should NodePort be in backticks? I have seen it written both ways in the docs.

Reply:
Yeah this part was a bit confusing when writing the main docs. In my understanding, its in backtics when referenincg the NodePort as a variable  But when writing it as "NodePort service" in sentence format, its without. Thats what is like in the Kubernetes docs as well.